.github/workflows/continous-integration.yaml

```diff
@@ -0,0 +1,32 @@
+name: Continous Integration
+
+# This workflow is triggered on pushes to the repository.
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
+          - macos-latest
+        node-version: [10, 12, 13]
```

Comment:
I feel it's redundant to test all Node versions on all platforms, I am concerned it will make builds take too long to finish. Can we please replicate the same matrix we have now?

- Linux: Node 10, 12, 13
- MacOS: Node 12
- Windows: Node 12 (upgrade from current AppVeyor config using Node 10)